core/chains/evm/forwarders/forwarder.go

```diff
@@ -0,0 +1,17 @@
+package forwarders
+
+import (
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/smartcontractkit/chainlink/core/utils"
+)
+
+// Forwarder is the external interface for ForwarderAddresses
+type EVMForwarder struct {
```

Comment:
That looks like a struct not an interface?

Reply:
ouch, fixed.